src/rust/en.reaperscans/src/parser.rs

```diff
@@ -0,0 +1,413 @@
+use aidoku::{
+	error::Result,
+	prelude::format,
+	std::html::Node,
+	std::net::{HttpMethod, Request},
+	std::{String, Vec},
+	Chapter, Listing, Manga, MangaContentRating, MangaPageResult, MangaStatus, MangaViewer, Page,
+};
+
+use crate::helper::*;
+use crate::request_helper::*;
+
+pub fn parse_manga_list(base_url: String, page: i32) -> Result<MangaPageResult> {
+	let url = format!("{}/comics?page={}", base_url, page);
+
+	let html = Request::new(url, HttpMethod::Get)
+		.header("User-Agent", &get_user_agent())
+		.html()
+		.expect(
+		"ReaperScans: Could not display All listing. Check the website and your internet connection on https://reaperscans.com",
+		);
+
+	let mut mangas: Vec<Manga> = Vec::new();
+
+	for manga in html.select("main div[wire:id] div > li > div").array() {
+		let manga_node = manga
+			.as_node()
+			.expect("Reaperscans: Failed to parse a manga node. Title could not be displayed");
+		let id = get_manga_id(manga_node.select("a").attr("href").read());
+		let cover = manga_node.select("img").attr("src").read();
+		let title = String::from(manga_node.select("a.text-sm").text().read().trim());
+		let url = manga_node.select("a").attr("href").read();
+
+		mangas.push(Manga {
+			id,
+			cover,
+			title,
+			author: String::new(),
+			artist: String::new(),
+			description: String::new(),
```

Comment:
use `..Default::default()` at the end of the struct for filling in unused values